network/omaha/protocol.go

```diff
@@ -253,7 +253,7 @@ type Action struct {
 	DisplayVersion        string `xml:"DisplayVersion,attr,omitempty"`
 	Sha256                string `xml:"sha256,attr,omitempty"`
 	NeedsAdmin            bool   `xml:"needsadmin,attr,omitempty"`
-	IsDeltaPayload        bool   `xml:"IsDeltaPayload,attr,omitempty"`
+	IsDelta               bool   `xml:"IsDelta,attr,omitempty"`
```

Comment:
Yeah, the response from roller is wrong but it hasn't mattered because we don't do deltas right now. When we get around to implementing deltas in roller we should update `go-omaha` based on my revised implementation here in mantle and port roller to it.